Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,9 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner(){
+    return(
+        <img className='banner' src='Day-I-Social-Card\social-card\public\favicon.ico'></img>
```

Comment:
The `src` here is questionable. A hosted link is in the [readme](https://github.com/LambdaSchool/React-UI-Components#day-1-mvp-requirements).